fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -346,6 +346,28 @@ ACTOR Future<Void> getRate(UID myID,
 	}
 }
 
+void proxyGRVThresholdExceeded(const GetReadVersionRequest* req, GrvProxyStats* stats) {
+	++stats->txnRequestErrors;
+	// FIXME: send an error instead of giving an unreadable version when the client can support the error:
+	// req.reply.sendError(proxy_memory_limit_exceeded());
+	GetReadVersionReply rep;
+	rep.version = 1;
+	rep.locked = true;
+	req->reply.send(rep);
+	if (req->priority > TransactionPriority::IMMEDIATE) {
```

Comment:
There are a few places using `IMMEDIATE` that I think should be default (or that should have their inequality adjusted).

```suggestion
	if (req->priority == TransactionPriority::IMMEDIATE) {
```